pkg/apis/v2v/v1alpha1/virtualmachineimport_types.go

```diff
@@ -90,7 +90,7 @@ type VirtualMachineImportStatus struct {
 	Conditions []VirtualMachineImportCondition `json:"conditions"`
 
 	// +optional
-	DataVolumes []DataVolumeItem `json:"dataVolumes"`
+	DataVolumes []DataVolumeItem `json:"dataVolumes,omitempty"`
```

Comment:
Is this relevant to updating conditions?

Reply:
Kind of; without it the update on status without this slice set fails API-level validation - at the early stages of import processing GET returns it set to nil.